Blaise-Shyaka/Politico
# [finishes #171722224] Create an endpoint to view all offices

# What does this PR do?

It creates an API endpoint allowing users to view all political offices

# Description of the tasks to be completed

- Write tests for the `GET /offices`
- Write a controller for that endpoint and mount it on a router
- Ensure that all the tests pass

# How can this be manually tested?

- Clone this repository
- Run `npm install` and `npm run test` to install all dependencies and run tests
- All the tests should pass

# What are the relevant pivotal tracker stories?

#171722224
